app/models/job.js

```diff
@@ -54,6 +54,7 @@ export default DS.Model.extend({
   lastJobError: Ember.computed('jobErrors.[]', function() {
     return this.get('jobErrors').sortBy('created').get('lastObject');
   }),
+  usage: DS.attr(),
```

Comment:
So wait a minute, we can just do `DS.attr()` and handle a boring old JS object from Ember Data without having to encode/decode JSON strings or architect relationships?

That...would have saved us some time 😆 . I like that flexibility for objects that don't need a persistent model, but I'm wondering if something like a [transform](https://guides.emberjs.com/release/models/defining-models/#toc_custom-transforms) would be good to encourage an expected structure.